Makefile

```diff
@@ -38,14 +38,34 @@ fmt: $(SOURCES_NO_VENDOR)
 	go run github.com/daixiang0/gci -w $^
 
 bin/$(GOOS)/influx: $(SOURCES)
-	$(GO_BUILD) -o $@ ./cmd/$(shell basename "$@")
+	CGO_ENABLED=0 $(GO_BUILD) -o $@ ./cmd/$(shell basename "$@")
 
 .DEFAULT_GOAL := influx
 influx: bin/$(GOOS)/influx
 
 vendor: go.mod go.sum
 	go mod vendor
 
+GORELEASER_VERSION := v0.165.0
```

Comment:
Is there a process in place to keep this updated?

Reply:
Not really... my thought was to keep things stable and only upgrade when we wanted new features, but if we want to always grab the latest-and-greatest when we could omit the `VERSION` tag entirely and have the download script find & get the latest available each time it runs